website/docs/docs/core/connect-data-platform/cratedb-setup.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "CrateDB setup"
+description: "Read this guide to learn about the CrateDB warehouse setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Community
+  authors: 'CrateDB maintainers'
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'n/a'
+  slack_channel_link: 'https://community.cratedb.com/'
+  platform_name: 'CrateDB'
+  config_page: '/reference/resource-configs/postgres-configs'
+---
+
+:::info Community plugin
+
+Some core functionality may be limited.
+
+:::
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta}/>
+
+
+## Profile Configuration
+
+CrateDB targets should be set up using the following configuration in your `profiles.yml` file.
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+company-name:
+  target: dev
+  outputs:
+    dev:
+      type: cratedb
+      host: [hostname]
+      user: [username]
+      password: [password]
+      port: [port]
+      dbname: [database name] # or database instead of dbname
+      schema: [dbt schema]
+      threads: [optional, 1 or more]
+      [keepalives_idle](#keepalives_idle): 0 # default 0, indicating the system default. See below
+      connect_timeout: 10 # default 10 seconds
+      [retries](#retries): 1  # default 1 retry on error/timeout when opening connections
+      [search_path](#search_path): [optional, override the default postgres search_path]
+      [role](#role): [optional, set the role dbt assumes when executing queries]
+      [sslmode](#sslmode): [optional, set the sslmode used to connect to the database]
+      [sslcert](#sslcert): [optional, set the sslcert to control the certifcate file location]
+      [sslkey](#sslkey): [optional, set the sslkey to control the location of the private key]
+      [sslrootcert](#sslrootcert): [optional, set the sslrootcert config value to a new file path in order to customize the file location that contain root certificates]
+  
+```
```

Comment:
That's the vanilla section of connection options copied from `postgres-setup.md`. AlloyDB, Greenplum and Yellowbrick omit some options they do not support. We should probably do the same.

Reply:
I've trimmed this section to include only a minimal set of parameters.